Owncloud iOs Client/InstantUpload/InstantUpload.m

```diff
@@ -0,0 +1,215 @@
+//
+//  InstantUpload.m
+//  Owncloud iOs Client
```

Comment:
@Jon-Schneider as @javiergonzper said you can leave your name as created by, but you must added the next OC copyright in all the new files
 /*
- Copyright (C) 2016, ownCloud, Inc.
- This code is covered by the GNU Public License Version 3.
- For distribution utilizing Apple mechanisms please see https://owncloud.org/contribute/iOS-license-exception/
- You should have received a copy of this license
- along with this program. If not, see <http://www.gnu.org/licenses/gpl-3.0.en.html>.
- */